src/scripts/SpellHandlers/Scripts/Priest.cpp

```diff
@@ -29,39 +29,29 @@ enum PriestSpells
     SPELL_HOLY_CONCENTRATION_R3             = 63725,
     SPELL_IMPROVED_DEVOURING_PLAGUE_R1      = 63625,
     SPELL_IMPROVED_DEVOURING_PLAGUE_R2      = 63626,
-#if VERSION_STRING == WotLK
     SPELL_IMPROVED_DEVOURING_PLAGUE_R3      = 63627,
-#endif
     SPELL_IMPROVED_DEVOURING_PLAGUE_DMG     = 63675,
     SPELL_IMPROVED_DEVOURING_PLAGUE_HEAL    = 75999,
     SPELL_IMPROVED_MIND_BLAST_R1            = 15273,
     SPELL_IMPROVED_MIND_BLAST_R2            = 15312,
     SPELL_IMPROVED_MIND_BLAST_R3            = 15313,
-#if VERSION_STRING < Cata
     SPELL_IMPROVED_MIND_BLAST_R4            = 15314,
     SPELL_IMPROVED_MIND_BLAST_R5            = 15316,
```

Comment:
![Captura de Pantalla 2021-05-15 a la(s) 02 18 54](https://user-images.githubusercontent.com/3104815/118351883-1292e900-b524-11eb-9917-e4a0b9ea55be.png)

http://www.cavernoftime.com/spell=15273

for example this spell only has 3 variants for cataclysm.